internal/service/timestreaminfluxdb/db_instance.go

```diff
@@ -239,64 +151,51 @@ func (r *resourceDbInstance) Schema(ctx context.Context, req resource.SchemaRequ
 						"",
 					),
 				},
+				Description: `The id of the DB parameter group assigned to your DB instance.`,
 			},
 			"db_storage_type": schema.StringAttribute{
 				Optional: true,
 				Computed: true,
+				Default:  stringdefault.StaticString(string(awstypes.DbStorageTypeInfluxIoIncludedT1)),
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.RequiresReplace(),
 				},
 				Validators: []validator.String{
 					stringvalidator.OneOf(
-						"InfluxIOIncludedT1",
-						"InfluxIOIncludedT2",
-						"InfluxIOIncludedT3",
+						string(awstypes.DbStorageTypeInfluxIoIncludedT1),
+						string(awstypes.DbStorageTypeInfluxIoIncludedT2),
+						string(awstypes.DbStorageTypeInfluxIoIncludedT3),
 					),
 				},
 				Description: `The Timestream for InfluxDB DB storage type to read and write InfluxDB data. 
 					You can choose between 3 different types of provisioned Influx IOPS included storage according 
 					to your workloads requirements: Influx IO Included 3000 IOPS, Influx IO Included 12000 IOPS, 
-					Influx IO Included 16000 IOPS. This argument is optional. Possible values: 
-					"InfluxIOIncludedT1", 
-					"InfluxIOIncludedT2", 
-					"InfluxIOIncludedT3". 
-					If not provided, defaults to "InfluxIOIncludedT1"`,
+					Influx IO Included 16000 IOPS.`,
 			},
 			"deployment_type": schema.StringAttribute{
 				Optional: true,
 				Computed: true,
+				Default:  stringdefault.StaticString(string(awstypes.DeploymentTypeSingleAz)),
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.RequiresReplace(),
 				},
 				Validators: []validator.String{
 					stringvalidator.OneOf(
-						"SINGLE_AZ",
-						"WITH_MULTIAZ_STANDBY",
+						string(awstypes.DeploymentTypeSingleAz),
+						string(awstypes.DeploymentTypeWithMultiazStandby),
 					),
 				},
 				Description: `Specifies whether the DB instance will be deployed as a standalone instance or 
-					with a Multi-AZ standby for high availability. This argument is optional. 
-					Possible values: 
-					"SINGLE_AZ", 
-					"WITH_MULTIAZ_STANDBY". 
-					If not provided, defaults to "SINGLE_AZ"`,
+					with a Multi-AZ standby for high availability.`,
 			},
 			"endpoint": schema.StringAttribute{
-				Computed: true,
+				Computed:    true,
+				Description: `The endpoint used to connect to InfluxDB. The default InfluxDB port is 8086.`,
```

Comment:
port 8086 is the only port that is supported.

Reply:
This is the description from AWS SDK Go. Link: https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/timestreaminfluxdb@v1.0.5#CreateDbInstanceOutput